components/brave_today/common/pref_names.cc

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/brave_today/common/pref_names.h"
+
+#include "components/prefs/pref_service.h"
+
+namespace brave_news {
+bool GetIsEnabled(PrefService* prefs) {
```

Comment:
this function should be in `brave_today/browser`.

Reply:
Changed, but why does it belong in `browser`? Is it because it depends on `components/prefs`?